build.rs

```diff
@@ -212,7 +212,18 @@ fn which_freebsd() -> Option<i32> {
 }
 
 fn emcc_version_code() -> Option<u64> {
-    let output = Command::new("emcc").arg("-dumpversion").output().ok()?;
+    let output = if cfg!(target_os = "windows") {
+        std::process::Command::new("emcc.bat")
+            .arg("-dumpversion")
+            .output()
+            .ok()
+    } else {
+        std::process::Command::new("emcc")
+            .arg("-dumpversion")
+            .output()
+            .ok()
+    }?;
```

Comment:
Actually this could be cleaned up a bit, we already have `Command` imported and the two commands are identical except for the executable name:

```rust
    let emcc = if cfg!(target_os = "windows") {
        "emcc.bat"
    } else {
        "emcc"
    };

    Command::new(emcc).arg("-dumpversion").output()?;
```